android/src/main/kotlin/net/mullvad/mullvadvpn/model/Settings.kt

```diff
@@ -1,10 +1,14 @@
 package net.mullvad.mullvadvpn.model
 
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
 data class Settings(
```

Comment:
Maybe name is not clear. Is it user settings or vpn settings or global settings?